README.md

```diff
@@ -48,8 +48,17 @@ Although the board shown in this examples is K64F, the example should work on an
 
  7. The serial console should now display a series of results following the NVStore API invocations. 
  
- 8. If you run this example on the K64F board, copy `mbed_app-K64F-8KB-areas.json` to `mbed_app.json`. This should use 8KB areas (two pairs of last sectors) as NVStore areas. If you run it on another board, the last sectors may have different addresses and sizes comparing to the K64F ones. You can copy the K64F NVstore configuration to your board's configuration. Then edit the configuration to use two pairs of the last board's sectors for NVStore.
-
+ 8. Unless specifically configured by the user, NVStore selects the last two flash sectors as its areas, with the minimum size of 4KBs,
+    meaning that if the sectors are smaller, a few continuous ones will be used for each area.
+    In case the automatically selected sectors do not fit your flash configuration, you can override this by setting the addresses and sizes of both areas in `mbed_lib.json` for each board. 
```

Comment:
If you wanted line breaks, you'll need to add another line (and indent so that it fits into the step).

Reply:
"In case" = "If".